locksmith/src/controllers/v2/verifierController.ts

```diff
@@ -156,4 +173,49 @@ export class VerifierController {
       })
     }
   }
+
+  // check is address is a Verifier of a specific lock
+  async isVerifierEnabled(request: Request, response: Response) {
+    try {
+      const lockAddress = Normalizer.ethereumAddress(request.params.lockAddress)
+      const address = Normalizer.ethereumAddress(request.params.verifierAddress)
+      const network = Number(request.params.network)
+
+      const verifierMatchesLoggedUser = request.user?.walletAddress === address
```

Comment:
That is not right (I think my comment was misleading!). We need to verify that `request.user?.walletAddress` is a lock manager on the lock...